templates/details_publication_collection.html

```diff
@@ -0,0 +1,59 @@
+{% extends "details_base.html" %}
+{% load markdown %}
+{% load i18n %}
+
+{% block extra_details %}
+
+  <div class="govuk-grid-row">
```

Comment:
Is this similar enough to databases/dashboards that we can extract a common partial? (We don't necessarily have to do it now, we could come back to it later if you prefer)

Reply:
raised [issue](https://github.com/ministryofjustice/find-moj-data/issues/1117)